### PR TITLE
add authentication instructions to yarn publish

### PR DESCRIPTION
**Pull request by vernak2539**
### Why:

Additional documentation is required to ensure publishing packages with the latest versions Yarn via GitHub actions succeeds.

I stumbled upon this when trying to use Yarn v4 to publish [my package](https://github.com/vernak2539/astro-rehype-relative-markdown-links) using a GitHub action. 

Only after adding [this line](https://github.com/vernak2539/astro-rehype-relative-markdown-links/blob/main/.yarnrc.yml#L3) did the publish complete correctly.

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Documentation should say it all. Happy to provide more info if necessary

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Review by devlopersabbir — COMMENTED** [review]
looks like cool

**Comment by nguyenalex836** [comment]
@vernak2539 Thanks so much for opening a PR! I'll get this triaged for review ✨

**Comment by vernak2539** [comment]
@nguyenalex836 has there been any progress on this by chance?

**Comment by nguyenalex836** [comment]
@vernak2539 Hello👋 ! Our team is still in the review process, but we will let you know as soon as review is complete. Thank you for your patience 💛

File: content/actions/publishing-packages/publishing-nodejs-packages.md [hunk 1]
```diff
@@ -180,3 +180,9 @@ jobs:
         env:
           NODE_AUTH_TOKEN: {% raw %}${{ secrets.NPM_TOKEN }}{% endraw %}
 ```
+
+To ensure authentication with the registry during publishing add a `npmAuthToken` entry to the `.yarnrc.yml`.
+
+```yaml
+npmAuthToken: '${NODE_AUTH_TOKEN:-}'
+```
```
Comment (SiaraMist) [inline content/actions/publishing-packages/publishing-nodejs-packages.md:188]:
```suggestion

To authenticate with the registry during publishing, ensure your authentication token is also defined in your `yarnrc.yml` file. For more information, see the [Settings](https://yarnpkg.com/configuration/yarnrc#npmAuthToken) article in the Yarn documentation.
```
I think this is very useful information to call out! However, I am hesitant to document the exact Yarn syntax in the GitHub docs. If Yarn updates the syntax they use in their `.yarnrc` file (like they did when they replaced `_authToken` with `npmAuthToken`), this doc will contain incorrect information and be out of date. I'd rather link to the Yarn docs so people can get the authentication instructions directly from them.

**Review by SiaraMist — DISMISSED** [review]
Hey @vernak2539, thank you so much for opening this pull request! ✨ 

I think this is definitely useful information to call out in this doc. I left one suggestion to update the phrasing to be a little more general and link to the Yarn docs. I'm going to go ahead and commit my suggestion and get this merged! Thank you again for catching this and taking the time to update our docs. 💛

**Review by SiaraMist — APPROVED** [review]

**Pull request merged**